multi-agent/lib/mas/elements/retrievers/slack/slack_retriever.py

```
@@ -37,7 +38,11 @@ def retrieve(self, query: str) -> Any:
             "loggedInUser": user_id,
         }
 
-        resp = requests.get(self.api_url, params=params)
+        resp = requests.get(
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)